app/src/androidTest/java/ch/epfl/qedit/HomePopUpTest.java

```diff
@@ -1,4 +1,5 @@
-/*package ch.epfl.qedit;
```

Comment:
Why is this entire class commented?

Reply:
Because otherwise the coverage is above 80% -_-, just kidding. I think Cosme didn't write the tests because he knew I would in my PR....

Reply:
Never mind, Antoine told me why.